README.md

```diff
@@ -1,2 +1,10 @@
 # fvm-utils
-This repo contains a series of crates that serve as utils for fvm develop and testing.
\ No newline at end of file
+This repo contains a series of crates that serve as utils for fvm develop and testing.
+Here are some simple breakdowns:
+- runtime: Contains the runtime wrapper for communicating with `fvm`. It provides some 
```

Comment:
TODO: Add an extended description of the runtime. Do you mind adding an issue to remind ourselves to do it in the future (I can probably take this one)

Reply:
created, https://github.com/consensus-shipyard/fvm-utils/issues/4